newIDE/app/src/VariablesList/styles.js

```diff
@@ -1,10 +1,10 @@
 export default {
   toolColumnHeader: {
-    textAlign: 'right',
-    paddingRight: 8,
+    textAlign: 'left',
+    paddingRight: 4,
   },
   toolColumn: {
-    minWidth: 48,
```

Comment:
Why changing this? This made the icon go on the area of the field:
![image](https://user-images.githubusercontent.com/1280130/53045025-e4c9b180-348c-11e9-9d35-5d1126b32c11.png)

Reply:
makes sense. Sorry I missed it